executor/child/query.go

```diff
@@ -33,14 +33,30 @@ func (ch Child) QueryWithdrawal(sequence uint64) (executortypes.QueryWithdrawalR
 		BridgeId:         ch.BridgeId(),
 		OutputIndex:      outputIndex,
 		WithdrawalProofs: proofs,
-		Sender:           withdrawal.From,
+		From:             withdrawal.From,
+		To:               withdrawal.To,
 		Sequence:         sequence,
-		Amount:           amount.String(),
+		Amount:           amount,
 		Version:          []byte{ch.Version()},
 		StorageRoot:      outputRoot,
-		LatestBlockHash:  treeExtraData.BlockHash,
-		BlockNumber:      treeExtraData.BlockNumber,
-		Receiver:         withdrawal.To,
-		WithdrawalHash:   withdrawal.WithdrawalHash,
+		LastBlockHash:    treeExtraData.BlockHash,
+		// BlockNumber:      treeExtraData.BlockNumber,
```

Comment:
is it okay to be skipped?

Reply:
It doesn't need it 
https://github.com/initia-labs/OPinit/blob/3965ef448c24e90049f138558fd171e62af82d23/x/ophost/types/tx.pb.go#L429